schematic/store/synapse.py

```diff
@@ -994,7 +994,7 @@ def associateMetadataWithFiles(
             load_args={
                 "dtype":"string",
             }
-            manifest = load_df(metadataManifestPath, preserve_raw_input=False, **load_args)
+            manifest = load_df(metadataManifestPath, preserve_raw_input = True, **load_args) #HOTFIX: set preserve_raw_input to true to allow mixed type cols in table uploads schematic#870
```

Comment:
@GiaJordan in the past we've had this set to False, if I remember right, because setting it to True could potentially cause errors in other parts of the code? Do I remember correctly? Or perhaps we decided to expose the parameter as an option, and we just thought False would be a good backward compatible value, but True is a good working option as well.

Reply:
@GiaJordan I am wondering the same as Milen. Also could you add a comment in function `load_df` to clarify parameter `preserve_raw_input`? Thank you!

Reply:
Thanks @GiaJordan!

Reply:
After doing manual tests, this fix doesn't appear to break `int` and `list` annotation uploads @milen-sage